### PR TITLE
add support usb bridged drive

### PR DESCRIPTION
**Pull request by tokohito**
Please add a function to support externally connected USB drives.
I'm sorry, but I have only checked the operation with 'usbprolific'.

Referenced information
https://www.smartmontools.org/wiki/USB

### CODE REVIEW

File: smartprom.py [hunk 1]
```diff
@@ -164,9 +164,9 @@ def collect():
     for drive, drive_attrs in DRIVES.items():
         typ = drive_attrs['type']
         try:
-            if typ == 'sat':
+            if typ == 'sat' or typ == 'usbjmicron' or typ == 'usbprolific' or typ == 'usbsunplus':
```
Comment (ngosang) [inline smartprom.py:167]: Could you change the code to something like this?
```
sat_types = ['sat', 'usbjmicron', 'usbprolific', 'usbsunplus']
nvme_types = ...
if typ in sat_types:
 ...
```

File: smartprom.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ def get_smart_status(results: dict) -> int:
 
 def smart_sat(dev: str) -> dict:
     """
-    Runs the smartctl command on a "sat" device
+    Runs the smartctl command on a internal or external "sat" device
     and processes its attributes
     """
     results, exit_code = run_smartctl_cmd(['smartctl', '-A', '-H', '-d', 'sat', '--json=c', dev])
```
Comment (ngosang) [inline smartprom.py:80]: Are you sure it works with the 'sat' type in the smarctl command instead of 'usbprolific'?

**Review by ngosang — CHANGES_REQUESTED** [review]
Thank you! I'm happy to merge but I let you 2 comments first.

File: smartprom.py (hunk 1 above)
Comment (tokohito) [reply smartprom.py:167]: Yes, of course.

File: smartprom.py (hunk 2 above)
Comment (tokohito) [reply smartprom.py:80]: It works for both 'sat' and 'usbprolific'.
Below is the command execution result in my environment, the output result of 'sat' and 'usbprolific' is the same.

> $ smartctl --scan
> /dev/sdd -d usbprolific # /dev/sdd [USB Prolific], ATA device
> $ sudo smartctl -A -d sat /dev/sdd
> $ sudo smartctl -A -d usbprolific /dev/sdd

**Comment by ngosang** [comment]
Thank you!

File: smartprom.py (hunk 1 above)
Comment (ngosang) [reply smartprom.py:167]: I will do it in another PR.

**Comment by ngosang** [comment]
https://github.com/matusnovak/prometheus-smartctl/releases/tag/v2.2.0

**Pull request merged**